helloworld/main.py

```diff
@@ -1,4 +1,62 @@
-from flask import escape
+import os
+import io
+from flask import escape, request
+from flask_restful import Resource, Api
+from google.cloud import vision
+
+from flask import Flask, jsonify
+from flask_cors import CORS
+
+os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = r'GVServiceAccountToken.json'
+
+app = Flask(__name__)
+api = Api(app)
+
+# Set this to false when deploying to live application
+app.config['DEBUG'] = True
+
+# Allow cross-origin resource sharing
+cors = CORS(app, resources={r'/api/*': {'origins': '*'}})
```

Comment:
why do we need this?

Reply:
We do not. It is just there. I'll get rid of everything unnecessary once I know I can deploy a function.